src/transactions/reducer.ts

```diff
@@ -22,11 +22,13 @@ interface State {
   // feed instantly.
   standbyTransactions: StandbyTransaction[]
   transactionsByNetworkId: TransactionsByNetworkId
+  knownCompletedTransactionsHashes: string[]
```

Comment:
Could we use a new `transactions` field to store the first page (or last X newest items) from the new API?
Then we could avoid the need for `knownCompletedTransactionsHashes` and use selectors to derive what we need from it.

Then if we see a new transaction that's newer than the oldest item in the newest list of items, we can vibrate.

This would address this new field proposed here being an ever growing list of TX hashes. And would serve the purpose of showing cached data directly on app load.

Would this all work?